src/themes/light.scss

```diff
@@ -38,6 +38,8 @@
   --wxc-banner--bottom--background: #{$wxc-white-alpha-70};
 
   --wxc-icon--background-color: #{$wxc-white-alpha-100};
+  --wxc-meeting-box-shadow: 0px 24px 48px #{$wxc-black-alpha-20}, 0px 0px 1px #{$wxc-black-alpha-12};
```

Comment:
no "meeting" variables please

maybe it's a primary box shadow? and please use rem instead of px